MANIFEST.in

```diff
@@ -0,0 +1,37 @@
+recursive-include unionml/templates *
+
+include README.md
+include src/pythae/py.typed  # marker file for PEP 561
+
+include CHANGELOG.md
+include changelog.md
+include LICENSE
+include LICENSE.txt
+include CITATION.cff
+include *.cff  # citation info
+
+include MANIFEST.in
+include src/pythae/pyproject.toml
+include setup.py
+include setup.cfg
+include tox.ini
+
+include requirements.txt
+
+
+recursive-exclude tests *
+recursive-exclude docs *
+recursive-exclude examples *
+recursive-exclude site *
+recursive-exclude .github *
+recursive-exclude .archives *
```

Comment:
No such file in repo

Reply:
By default those are located in a `dist` folder so maybe we can change the `.archives` with `dist`.

Reply:
changed.